website/content/docs/k8s/installation/vault/systems-integration.mdx

```diff
@@ -0,0 +1,168 @@
+---
+layout: docs
+page_title: Vault as Secrets Backend Systems Integration Overview
+description: >-
+  Overview of the systems integration aspects to using Vault as secrets backend for Consul on Kubernetes.
+---
+
+# Vault as the secrets backend - Systems Integration
+
+## Overview
+At a high level, configuring a systems integration of Vault with Consul on Kubernetes consists of the following steps:
+- One time set up on Vault
+  - Enabling Vault KV Secrets Engine - Version 2 to store arbitrary secrets
+  - Enabling Vault PKI Engine if you are choosing to store and manage either [Consul Server TLS credentials](/docs/k8s/installation/vault/data-integration/server-tls) or [Service Mesh and Consul client TLS credentials](/docs/k8s/installation/vault/data-integration/connect-ca)
+- Set up per Consul data center
+  - Installing the Vault Injector within the Consul data center installation
+  - Configuring a Kubernetes Auth Method in Vault to authenticate and authorize operations from the Consul data center
+  - Enable Vault as the secrets backend in the Consul data center
+
+## One time set up on Vault
+### Vault KV Secrets Engine - Version 2
+The following secrets can be stored in Vault KV secrets engine, which is meant to handle arbitrary secrets:
+- ACL Bootstrap token ([`global.acls.bootstrapToken`](/docs/k8s/helm#v-global-acls-bootstraptoken))
+- Gossip encryption key ([`global.gossipEncryption`]())
+- Enterprise license ([`global.enterpriseLicense]())
+- ACL Replication token ([`global.acls.replicationToken`](/docs/k8s/helm#v-global-acls-replicationtoken))
+- Snapshot Agent config ([`client.snapshotAgent.configSecret`](/docs/k8s/helm#v-client-snapshotagent-configsecret))
+
+In order to store any of these secrets, we must enable the [Vault KV secrets engine - Version 2](https://www.vaultproject.io/docs/secrets/kv/kv-v2).
+
+```shell-session
+$ vault secrets enable -path=consul kv-v2
+```
+
+### Vault PKI Engine
+
+The Vault PKI Engine must be enabled in order to leverage Vault for issuing Consul Server TLS certificates. More details for configuring the PKI Engine is found in [Bootstrapping the PKI Engine](/docs/k8s/installation/vault/data-integration/server-tls#bootstrapping-the-pki-engine) under the Server TLS section.
+
+```shell-session
+$ vault secrets enable pki
+```
+
+## Set up per Consul data center
```

Comment:
```suggestion
## Set up per Consul Datacenter
```

Datacenter is a proper noun for Consul representing a cluster deployed within a region.